### PR TITLE
Add inputProps type definition - react-tag-input 6.6

### PR DESCRIPTION
**Pull request by michal-skladanowski**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes:  https://github.com/react-tags/react-tags/blob/v6.6.0/src/components/ReactTags.js
PR adding the change: https://github.com/react-tags/react-tags/pull/726
commit adding the change: https://github.com/react-tags/react-tags/pull/726/commits/769d70641fb1f3236164f6e3049c2af84ac891b3
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## react-tag-input ([<kbd>unpkg</kbd>](https://unpkg.com/browse/react-tag-input@latest/))
was missing the following properties:
1. KEYS
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against b623e7947dce1efec424509a0068b7e0d22afaa9
</p>

**Comment by peterblazejewicz** [comment]
please add the url to a commit adding a change, thx!

**Comment by michal-skladanowski** [comment]
> please add the url to a commit adding a change, thx!

I've updated description with link to PR and specific commit.

File: types/react-tag-input/index.d.ts [hunk 1]
```diff
@@ -45,6 +45,7 @@ export interface ReactTagsProps {
 
     inline?: boolean | undefined;
     inputFieldPosition?: 'top' | 'bottom' | 'inline' | undefined;
+    inputProps?: React.InputHTMLAttributes<HTMLInputElement>;
```
Comment (peterblazejewicz) [inline types/react-tag-input/index.d.ts:48]:
```suggestion
    inputProps?: React.InputHTMLAttributes<HTMLInputElement> | undefined;
```
(to stay in sync with other, this is optional in props, supports 4.4 exact optional property types)

**Review by johnnyreilly — APPROVED** [review]

**Pull request merged**